inhibit/inhibit.go

```diff
@@ -167,22 +168,30 @@ type InhibitRule struct {
 // NewInhibitRule returns a new InhibitRule based on a configuration definition.
 func NewInhibitRule(cr *config.InhibitRule) *InhibitRule {
 	var (
-		sourcem types.Matchers
-		targetm types.Matchers
+		sourcem labels.Matchers
+		targetm labels.Matchers
 	)
 
 	for ln, lv := range cr.SourceMatch {
-		sourcem = append(sourcem, types.NewMatcher(model.LabelName(ln), lv))
+		sourcem = append(sourcem, &labels.Matcher{
+			Type:  labels.MatchEqual,
+			Name:  ln,
+			Value: lv,
+		})
 	}
 	for ln, lv := range cr.SourceMatchRE {
-		sourcem = append(sourcem, types.NewRegexMatcher(model.LabelName(ln), lv.Regexp))
+		sourcem = append(sourcem, labels.NewRegexpMatcher(ln, lv.Regexp))
 	}
 
 	for ln, lv := range cr.TargetMatch {
-		targetm = append(targetm, types.NewMatcher(model.LabelName(ln), lv))
+		targetm = append(targetm, &labels.Matcher{
+			Type:  labels.MatchEqual,
+			Name:  ln,
+			Value: lv,
+		})
```

Comment:
Just to be conservative, could you use `NewMatcher(labels.MatchEqual, ln, lv)` here? I am aware that this requires"handling" the error (just panic in case it err's out because that "must not happen" (TM)). But there might be changes to the labels.Matcher type in the future where more work is required in the constructor.